src/components/HeaderDrawer/index.js

```diff
@@ -21,21 +22,28 @@ const useStyles = makeStyles({})
 
 export default ({
   recentItems,
+  changeRecentItems,
   onClickHome,
   onCloseDrawer,
   drawerOpen,
   onOpenFile,
   onOpenRecentItem,
   onClickTemplate,
 }) => {
-  const c = useStyles()
-
+  const c = useStyles();
   const onDrop = useCallback((acceptedFiles) => {
     onOpenFile(acceptedFiles[0])
   }, [])
 
-  const { getRootProps, getInputProps } = useDropzone({ onDrop })
+  function onDeleteFile(i) {
+    console.log(i);
+    changeRecentItems( recentItems.filter(function(oneRecentFile){console.log(oneRecentFile.id);return oneRecentFile.id!==i}));
+    localStorage.removeItem('recentItems');
+    localStorage.setItem('recentItems',JSON.stringify(recentItems));
```

Comment:
this is implicitly handled in the `useLocalStorage` method, so you shouldn't need to interact directly with `localStorage` (no setItem/deleteItem required :)